eventyay_stripe/payment.py

```
@@ -539,10 +539,24 @@ def create_payment_intent(self, payment, event, payment_method_id, method,
+            if hasattr(payment.order, 'invoice_address') and payment.order.invoice_address:
+                ia = payment.order.invoice_address
+                base_params['shipping'] = {
+                    'name': ia.name or payment.order.email or 'Customer',
+                    'address': {
+                        'line1': ia.street or 'N/A',
+                        'city': ia.city or 'N/A',
+                        'postal_code': ia.zipcode or '00000',
+                        'country': getattr(ia.country, 'code', str(ia.country)) or 'IN',
+                    }
+                }
+        except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)